concat-on-disk.ipynb

```diff
@@ -0,0 +1,855 @@
+{
```

Comment:
<p>this uses a lot of memory, shouldn’t it be almost free?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>